src/homepage/src/screens/home/NPMFeature/SearchInput/index.js

```diff
@@ -0,0 +1,71 @@
+import React from 'react';
+import styled from 'styled-components';
+
+import algoliasearch from 'algoliasearch';
+import {
+  ALGOLIA_API_KEY,
+  ALGOLIA_APPLICATION_ID,
+  ALGOLIA_DEFAULT_INDEX,
+} from 'app/utils/config';
+
+import Input from './Input';
+import Hit from './Hit';
+
+import algoliaImage from './algolia.svg';
+import { searchFacets } from '../../../../utils/algolia';
+import media from '../../../../utils/media';
+
+const Legenda = styled.div`
```

Comment:
Dutch! 😬

Reply:
Hah, whoops! I let some Dutch slip in 😅 . Fixing.